types/address/hash_test.go

```diff
@@ -0,0 +1,90 @@
+package address
```

Comment:
Can we use test packages, please?

```suggestion
package address_test
```

Reply:
These are unit tests.